test/thermo/consistency.cpp

```diff
@@ -0,0 +1,691 @@
+#include "gtest/gtest.h"
```

Comment:
I think it would be good to add a header with a short description and a note indicating that tests are documented in `test/data/consistency-cases.yaml` (also, that relevant test loaders start below line 521).

It could also be argued that the yaml isn't cantera input data, but data that belongs to this googletest, and move the YAML into `test/thermo`.

Reply:
I did hesitate about putting the YAML fiile in `test/data`. The difficulty with putting it in `test/thermo` is that isn't on the data path when the test runs (the working directory is `build/test/thermo`), and you'd have to specify the input file using a somewhat fragile relative path.

Reply:
I think it’s fine leaving it where it is, but this makes the comment in the header more important.

Reply:
On second thought ... is there a simple way to have SCons copy YAML over from `test/thermo` to `build/test/thermo`? I think that would be the preferred solution. `test/data` (still) is a large folder where things aren't easily found.

Reply:
I'm of course fine with adding an explanatory comment to this file, but I'd rather not have to introduce an extra complication in `test/SConscript` that's really only used for this one case. Also, the more magic that we implement in `SCons`, the harder it is to reliably run tests outside of SCons, which is something that is occasionally necessary, for instance when you want to run the tests in `gdb`.